src/OpenLoco/src/Economy/Economy.cpp

```diff
@@ -78,6 +78,11 @@ namespace OpenLoco::Economy
         }
     }
 
+    currency32_t* getDeliveryCargoPaymentsTable(uint8_t cargoType)
```

Comment:
can't we return a proper type like `const std::array<currency32_t, 60>&`

Reply:
I've gone with a `std::span` instead. Seemed appropriate.